README.md

```diff
@@ -54,26 +54,26 @@ All HNC issues are assigned to an HNC milestone. So far, the following
 milestones are defined or planned:
 
 * v1.0 - Likely late Q2 or early Q3 2021: HNC recommended for production use
-* [v0.9](https://github.com/kubernetes-sigs/multi-tenancy/milestone/21): move
+* [v0.9](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/21): move
   HNC to its own repo; continued stability improvements.
-* [v0.8 - COMPLETE APR 2021](https://github.com/kubernetes-sigs/multi-tenancy/milestone/20):
+* [v0.8 - COMPLETE APR 2021](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/20):
   incremental stability improvements
-* [v0.7 - COMPLETE DEC 2020](https://github.com/kubernetes-sigs/multi-tenancy/milestone/18):
+* [v0.7 - COMPLETE DEC 2020](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/18):
   introduce exceptions.
-* [v0.6 - COMPLETE OCT 2020](https://github.com/kubernetes-sigs/multi-tenancy/milestone/14):
+* [v0.6 - COMPLETE OCT 2020](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/14):
   introduce the v1alpha2 API and fully automated end-to-end testing.
-* [v0.5 - COMPLETE JUL 2020](https://github.com/kubernetes-sigs/multi-tenancy/milestone/13):
+* [v0.5 - COMPLETE JUL 2020](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/13):
   feature simplification and improved testing and stability.
-* [v0.4 - COMPLETE JUN 2020](https://github.com/kubernetes-sigs/multi-tenancy/milestone/11):
+* [v0.4 - COMPLETE JUN 2020](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/11):
   stabilize the API and add productionization features.
-* [v0.3 - COMPLETE APR 2020](https://github.com/kubernetes-sigs/multi-tenancy/milestone/10):
+* [v0.3 - COMPLETE APR 2020](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/10):
   type configuration and better self-service namespace UX.
-* [v0.2 - COMPLETE DEC 2019](https://github.com/kubernetes-sigs/multi-tenancy/milestone/8):
+* [v0.2 - COMPLETE DEC 2019](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/8):
   contains enough functionality to be suitable for non-production workloads.
-* [v0.1 - COMPLETE NOV 2019](https://github.com/kubernetes-sigs/multi-tenancy/milestone/7):
+* [v0.1 - COMPLETE NOV 2019](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/7):
```

Comment:
All the completed milestones should probably stay in the old repo

Reply:
Ok sounds good, I figured we were trying to migrate the milestones/issues over so we'd have them here. Will change back

Reply:
Yeah I'd rather only move the active cases, especially since the ID numbers will all change IIUC.